buildspec.yml

```diff
@@ -0,0 +1,25 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - sudo apt-get update
+      - sudo apt-get install apt-transport-https
+      - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+      - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+      - sudo apt-get update && sudo apt-get install yarn
+  pre_build:
+    commands:
+      - yarn install
+  build:
+    commands:
+      - yarn build:all
+      - ./node_modules/mocha/bin/mocha -r ts-node/register test/**/*.spec.ts
+artifacts:
+  files:
+    - 'handler.js'
+  discard-paths: yes
+  base-directory: src/handlers
+cache:
+  paths:
+    - '/root/.npm/**/*'
```

Comment:
Not a blocker, but something that would speed this up in the future is having a tailor-made Docker image, that way we don't have to do all of the Debian install stuff.  We'll tackle that when the time comes, though.